project/AppDependencies.scala

```diff
@@ -6,19 +6,18 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-play-26"            % "2.1.0",
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"    % "2.25.0",
```

Comment:
For my own benefit, where does this exist? I see https://github.com/hmrc/bootstrap-play but don't see a 27 version

Reply:
@DevAndyroo It's that project. It allows you to pull in either "bootstrap-backend-play-27" or "bootstrap-backend-play-26"